src/sanitizer.js

```diff
@@ -286,6 +286,9 @@ function resolveAttrValue(tagName, attrName, attrValue) {
  * @private Visible for testing.
  */
 export function resolveUrlAttr(tagName, attrName, attrValue, windowLocation) {
+  user().assert(attrValue.indexOf('__amp_source_origin') == -1,
```

Comment:
I think this is fine, the other option is to import the constant defined in another file (since this string is being used in multiple files now) which might add more to the obfuscation. Not sure, @erwinmombay?